lib/hanami/action/glue.rb

```diff
@@ -59,7 +59,7 @@ def finish
       #
       # @since 0.4.3
       def sending_file?
-        @_body.is_a?(::Rack::File)
+        @_body.is_a?(::Rack::File) || @_body&.first.is_a?(::Rack::File)
```

Comment:
in this place you may get nil after `@_body&.first` and `undefined method `is_a' for nil:NilClass`